pepagent/const.py

```diff
@@ -4,6 +4,10 @@
 ANNO_COL = "anno_info"
 NAMESPACE_COL = "namespace"
 NAME_COL = "name"
+TAG_COL = "tag"
 DIGEST_COL = "digest"
 
-DB_COLUMNS = [ID_COL, PROJ_COL, ANNO_COL, NAMESPACE_COL, NAME_COL, DIGEST_COL]
+DB_COLUMNS = [ID_COL, DIGEST_COL, PROJ_COL, ANNO_COL, NAMESPACE_COL, NAME_COL, TAG_COL]
+
+DEFAULT_NAMESPACE = "other"
+DEFAULT_TAG = "primary"
```

Comment:
@nsheff what's a good default tag? `latest` or `primary`. I'm relatively agnostic about it all.

Reply:
I suggest `default`